content/en/architecture/component-overview.md

```diff
@@ -68,24 +71,20 @@ hide footbox
 participant D as "Device"
 participant CGW as "CoAP Gateway"
 participant AS as "Authorization Server"
-participant RA as "Resource Aggregate"
 
 D -> CGW ++: Sign Up
 group OAuth2.0 Authorization Code Grant Flow
-    CGW -> AS ++: Exchange authorization code for access token
-    AS -> AS: Verify Authorization Code
-    return Ok\n(Access Token, Refresh Token, ...)
+    CGW -> CGW: Verify and exchange authorization code for access token
 end
-CGW -> RA ++: Register device resource
-return Registered
+CGW -> AS ++: Add device resource
```

Comment:
CGW Registers device, or Register user's device

Reply:
This wasn’t resolve. Coap gw does not registers resource. It’s only device, creating record that device is connected and assigning it to a iser.

Reply:
Sorry, I've read this in the morning and forgot about this. Thanks for updating it.